Pipfile

```diff
@@ -13,6 +13,8 @@ ipython = "*"
 faker = "*"
 factory-boy = "*"
 pylint-django = "*"
+pandas = "*"
```

Comment:
Ok, I'm on the :panda_face: train now - it was _way_ nicer than my original approach with relying on Django, or using some CSV CLI tool